modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -198,6 +198,95 @@ class MirrordExecManager(private val service: MirrordProjectService) {
         return executionInfo
     }
 
+    private fun containerStart(
+        wslDistribution: WSLDistribution?,
+        product: String,
+        projectEnvVars: Map<String, String>?
+    ): MirrordContainerExecution? {
+        MirrordLogger.logger.debug("MirrordExecManager.startContainer")
+        val explicitlyEnabled = projectEnvVars?.any { (key, value) -> key == "MIRRORD_ACTIVE" && value == "1" } ?: false
+        if (!service.enabled && !explicitlyEnabled) {
```

Comment:
This does not respect `MIRRORD_ACTIVE=0`, does it?

Reply:
well it would fallback to false if it can't find `MIRRORD_ACTIVE=1` but this is existing logic, I just copied it over